apps/studio/build/deb-postinstall

```diff
@@ -23,10 +23,8 @@ if [ ! -f /usr/bin/beekeeper-studio ]; then
   ln -s /opt/Beekeeper\ Studio/beekeeper-studio /usr/bin/beekeeper-studio
```

Comment:
When testing these script changes I noticed that I get a warning from `ln` that the file already exists. I think the check should use `-e` instead of `-f`.

Reply:
oh, I guess /usr/bin/beekeeper-studio is a symlink now. Yes that will need to use `-e`. Good find! Feel free to update in this PR also.

Legend :star:

Reply:
Done